NOTES.md

```diff
@@ -0,0 +1,197 @@
+# User-defined claims
```

Comment:
I will move all of this content to GitHub discussion, as previously agreed. It can cross-link the relevant PR(s) where useful.